yb-voyager/cmd/liveMigrationReportCommand.go

```diff
@@ -0,0 +1,264 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/datafile"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/dbzm"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/metadb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+var liveMigrationReportCmd = &cobra.Command{
+	Use:   "report",
+	Short: "This command will print the report of any live migration workflow.",
+	Long:  `This command will print the report of the live migration or live migration with fall-forward or live migration with fall-back.`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		validateExportDirFlag()
+		migrationStatus, err := metaDB.GetMigrationStatusRecord()
+		if err != nil {
+			utils.ErrExit("error while getting migration status: %w\n", err)
+		}
+		streamChanges, err := checkWithStreamingMode()
+		if err != nil {
+			utils.ErrExit("error while checking streaming mode: %w\n", err)
+		}
+		migrationUUID, err = uuid.Parse(migrationStatus.MigrationUUID)
+		if err != nil {
+			utils.ErrExit("error while parsing migration UUID: %w\n", err)
+		}
+		if streamChanges {
+			getTargetPassword(cmd)
+			migrationStatus.TargetDBConf.Password = tconf.Password
+			if migrationStatus.FallForwardEnabled {
+				getFallForwardDBPassword(cmd)
+				migrationStatus.FallForwardDBConf.Password = tconf.Password
+			}
+			if migrationStatus.FallbackEnabled {
+				getSourceDBPassword(cmd)
+				migrationStatus.SourceDBAsTargetConf.Password = source.Password
+			}
+			liveMigrationStatusCmdFn(migrationStatus)
+		} else {
+			utils.ErrExit("export-type 'snapshot-and-changes' is not enabled for this migration")
```

Comment:
Maybe a more msg like `live-migration report is only applicable when export-type is 'snapshot-and-changes' ?